tableau_utilities/tableau_file/tableau_file.py

```diff
@@ -74,8 +77,16 @@ def unzip(self, unzip_all=False):
                     tableau_file_path = os.path.join(file_dir, z.filename)
         return tableau_file_path
 
-    def save(self):
+    def save(self, save_raw_xml=False):
```

Comment:
@JustinGrilli  do you by any change know an easy way to preserve the attribute order when I output the xml?  I'm trying to diff files before and after changes but I don't see a simple way to make that happen.

If you have a pointer on this I'll take it otherwise I can definitely slug through it.

Reply:
I believe there's a bunch of conversion happening which causes the attribute order to shift, i.e.
XML -> OrderedDict -> dataclass -> dict -> XML

I'm not sure that there is a way to maintain the attribute order for dataclasses, especially when converting from dataclass -> dict -> XML.
In order to maintain the attribute order, we'd have to probably ditch dataclasses.

Reply:
Something I just thought of, is we could take note of the order as we are reading it. Assign that order as an attribute of the class, and then enforce the order as the class is converting back to XML.

For example (pseudo code):
```python
@dataclass
class TableauFile:
    _attr_order: list = None

    def __post_init__():
        self._attr_order = [...the attribute order here...]
    
    def save():
        for attr in self._attr_order:
            # ... create OrderedDict in the order we need, then convert to OrderdDict -> XML
```